06-TimeManagement/modul/time_management.c

```diff
@@ -33,6 +33,27 @@ struct class_attribute attr_relation_time = {
 	.show	= relation_time_show,
 };
 
+
+static ssize_t absolute_time_show(struct class *class,
+	struct class_attribute *attr, char *buf)
+{
+	static struct timespec prev_ts;
+
+	pr_info("TimingManagement:PrevAbsolute time: %lu.%lu\n", prev_ts.tv_sec,
+		prev_ts.tv_nsec);
+	sprintf(buf, "TimingManagement:PrevAbsolute time: %lu.%lu\n",
```

Comment:
Error in format string.

Reply:
Fixed: sprintf(buf, "TimingManagement:PrevAbsolute time: %lu.%09lu\n",
		prev_ts.tv_sec, prev_ts.tv_nsec);